.drone.yml

```
@@ -1,120 +1,149 @@
 ---
 kind: pipeline
 type: kubernetes
-name: deploy
+name: deploy tag
+trigger:
+  event:
+    - tag
+  branch:
+    - main
+
+steps:
+  - name: checkout
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    commands:
+      - git fetch --tags
+      - git checkout ${DRONE_TAG}
+      - git submodule update --init --recursive
+
+  - &deploy
+    name: cs-dev
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    environment:
+      CHART_NAME: hocs-notify
+      KUBE_CLUSTER: acp-notprod
+      KUBE_NAMESPACE: cs-dev
```

If you did this:
```suggestion
      KUBE_NAMESPACE: ${DRONE_STEP_NAME}
```
You wouldn't need to overwrite `environment:` in the wcs-dev deploy later on.